### ISSUE TITLE
Products not saved

### ISSUE THREAD

**Issue by JaapWeijland** [pravinnavale/Assignment1#11]
https://github.com/pravinnavale/Assignment1/blob/ae56afc1b3e6547a7b0db6b3c72ace2140f46793/apps/src/getAllProducts.ts#L8

This seems to add the product to the temporary products array, which is deleted after execution of the route.

**Comment by pravinnavale** [comment]
Yes, temporary array is used just to filter response based on simulation day
